lib/modules/profile/screens/auth_gate.dart

```diff
@@ -16,9 +22,11 @@ class AuthGate extends StatelessWidget {
           // User is not signed in
           return const SignInScreen();
         }
-
-        final user = snapshot.data!;
-        return ProfileScreen(user);
+        if (openProfile) return ProfileScreen(snapshot.data!);
+        SchedulerBinding.instance?.addPostFrameCallback(
```

Comment:
This (and 3 other instances of `SchedulerBinding.instance`) trigger the `invalid_null_aware_operator` warning